rmd - readme - hw01.Rmd

```diff
@@ -0,0 +1,33 @@
+---
+title: "Bio-hw01"
+author: "Frank"
```

Comment:
You didn't need a rmarkdown file for this exercise! No need to overcomplicate things for this.